apps/E2E/src/Text/specs/Text.spec.win.ts

```diff
@@ -28,17 +32,39 @@ describe('Text Accessibility Testing', () => {
   });
 
   it('Text - Validate accessibilityRole is correct', async () => {
-    await expect(await TextPageObject.getAccessibilityRole()).toEqual(TEXT_A11Y_ROLE);
+    await expect(await TextPageObject.getTextAccessibilityRole(TextComponentSelector.First)).toEqual(TEXT_A11Y_ROLE);
     await expect(await TextPageObject.didAssertPopup()).toBeFalsy(TextPageObject.ERRORMESSAGE_ASSERT);
   });
 
   it('Text - Set accessibilityLabel', async () => {
-    await expect(await TextPageObject.getAccessibilityLabel(ComponentSelector.Primary)).toEqual(TEXT_ACCESSIBILITY_LABEL);
+    await expect(await TextPageObject.getTextAccessibilityLabel(TextComponentSelector.First)).toEqual(TEXT_ACCESSIBILITY_LABEL);
     await expect(await TextPageObject.didAssertPopup()).toBeFalsy(TextPageObject.ERRORMESSAGE_ASSERT);
   });
 
-  it('Text - Do not accessibilityLabel -> Default to content', async () => {
-    await expect(await TextPageObject.getAccessibilityLabel(ComponentSelector.Secondary)).toEqual(TEXT_COMPONENT_CONTENT);
+  it('Text - Do not set accessibilityLabel -> Default to content', async () => {
+    await expect(await TextPageObject.getTextAccessibilityLabel(TextComponentSelector.Second)).toEqual(TEXT_COMPONENT_CONTENT);
+    await expect(await TextPageObject.didAssertPopup()).toBeFalsy(TextPageObject.ERRORMESSAGE_ASSERT);
+  });
+});
+
+describe('Deprecated Text Accessibility Testing', () => {
```

Comment:
How is this `describe()` block different than the `describe()` block above? And how is it different than the ExperimentalText `describe()` block?

I'm a bit confused because I'd assume we'd only have two `describe('accessibility testing')` blocks spanning across the two files - one for deprecated and one for V1. But it seems that we have 3